source/__tests__/integration/export-profile.test.js

```diff
@@ -59,7 +60,6 @@ describe('Profiles Export', () => {
       await page.waitFor(1000, {waitUntil: 'networkidle2'})
       const valid_url_class = await page.$('input[name="resourceURI"]', e => e.getAttribute('ng-valid-url'))
       expect(valid_url_class).toBeTruthy()
-      return new Promise(resolve => resolve())
```

Comment:
I originally put this in thinking it had to end with a Promise to finish executing.  Taking it out seemed to have no ill effect, AFAICT.  YMMV.

Reply:
🤷‍♂